ui/pages/swaps/view-quote/view-quote.js

```diff
@@ -134,6 +150,8 @@ export default function ViewQuote() {
   const chainId = useSelector(getCurrentChainId);
   const nativeCurrencySymbol = useSelector(getNativeCurrency);
 
+  const gasFeeInputs = useGasFeeInputs('high');
```

Comment:
For Swaps we want to get 'high' estimations by default.

Reply:
Yep!